### ISSUE TITLE
UIR: Load testing

### ISSUE THREAD

**Issue by trackpete** [trackpete/exiletools-indexer#82]
When the report is complete, perform some basic load testing to ensure the system can handle initial expected load. Expect up to 100+ simultaneous requests.

**Comment by trackpete** [comment]
I created a list of ~130 URL's for various uniques and used a browser extension to load them all at the same time. The immediate problem was that the `DEVELOPMENT-Indexer` API Key got blocked as it exceeded the rate limit, so I created a new API Key for this front end (and since it's in the JS I'm trusting people not to use it, since I might expire it at any point).

Once that was done, the page loads were generally snappy. The overall performance was completely acceptable. The one thing I'm concerned about is that the browser may have been pipelining these connections, which would create a slightly different experience than actually creating 400+ connections to  the NGINX reverse proxy at the same time.

I need to check some browser settings and make sure this is a realistic test. I looked into using something like Selenium to do the tests but frankly it seemed way too complex for what I simply want to know - can it handle 100+ loads at the same time or not. 

Note that the backend pipeline was tested for this, including NGINX -> Tyk -> ES, but the front end load balancer was not included.
